hw2/03-count.js

```diff
@@ -1,3 +1,18 @@
 // Add your code here
 
-input.addEventListener('keydown', handleKeyDown);
+userInput.addEventListener("keydown", handleKeyDown);
+const elem = document.querySelector("input");
+
+function handleKeyDown(event) {
+  let paragraph = document.getElementById("text");
+  let textToSearch = elem.value;
+  console.log(textToSearch);
+  textToSearch = textToSearch.replace(/[.*+?^${}()|[|]\\]/g, "\\$&");
```

Comment:
it can search fraction of word. The requirement is to search for a word. partial word matched here.  when using backspace deleted previous word it still keep the last letter highlighted.

Reply:
I searched time and it highlited time